src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --blue-color:#00ACDC;
```

Comment:
The variable names should describe their purpose, if the designer changes the color to red it will be strange when the blue color variable does the color red. For example `--main-color`